tools/ci_build/build.py

```diff
@@ -1360,9 +1360,9 @@ def build_python_wheel(
             args.append('--use_nuphar')
         if wheel_name_suffix:
             args.append('--wheel_name_suffix={}'.format(wheel_name_suffix))
-        elif use_acl:
+        if use_acl:
```

Comment:
I think use_acl should be moved below use_nuphar as elif

Reply:
Reconciled the argument usage between `setup.py` and `build.py`, however `build.py` is passing `--use_acl` to `setup.py`, which isn't explicitly parsed in that file.